banderwagon/precomp.go

```diff
@@ -61,20 +61,24 @@ func NewPrecompMSM(points []Element) (MSMPrecomp, error) {
 // MSM calculates the 256-MSM of the given scalars on the fixed basis.
 // It automatically detects how many non-zero scalars there are and parallelizes the computation.
 func (msm *MSMPrecomp) MSM(scalars []fr.Element) Element {
-	result := Identity.inner
+	result := bandersnatch.IdentityExt
 
 	for i := range scalars {
 		if !scalars[i].IsZero() {
 			msm.precompPoints[i].ScalarMul(scalars[i], &result)
 		}
 	}
-	return Element{inner: result}
+	return Element{inner: bandersnatch.PointProj{
+		X: result.X,
+		Y: result.Y,
+		Z: result.Z,
+	}}
```

Comment:
After we computed the MSM, which returns a `PointExtended`, we create the projective point by just dropping `T`.